docker/build_docker.sh

```diff
@@ -105,9 +105,10 @@ function commit_dockerfiles_trust {
         git status --short
         git stash
         git pull --no-rebase
-        git checkout stash -- .
+        echo "done pulling"
+        git stash apply
```

Comment:
just make sure `stash apply` overrides the conflicts (if there are any) instead of asking you to handle a non trivial merge. I believe I used `checkout stash -- .` so it always overrides the conflicts.

Reply:
maybe we should do `git push --force` instead, in this case?